stl/inc/system_error

```diff
@@ -16,7 +16,8 @@
 #include <xerrc.h>
 #ifndef _M_CEE_PURE
 #include <atomic>
-#endif
+#include <compare>

```

Comment:
This will emit `"The contents of <compare> are available only with C++20 or later."` in C++14/17 mode. I believe this should be guarded with `#if _HAS_CXX20` instead of `#ifndef _M_CEE_PURE`.